<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>measureAreaDialog</class>
 <widget class="QDialog" name="measureAreaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Measure Parameter Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="CheckBox" name="baseInCartesianCb">
       <property name="text">
        <string>Based on Cartesian Crs</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">baseModeButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="CheckBox" name="baseInEllipsoidCb">
       <property name="text">
        <string>Based on Ellipsoid</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">baseModeButtonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <widget class="BodyLabel" name="BodyLabel_2">
       <property name="text">
        <string>Decimal Point Retained</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="SpinBox" name="decimalPrecisionSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BodyLabel" name="BodyLabel_3">
       <property name="text">
        <string>places</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CheckBox" name="autoTransUnitCheckBox">
     <property name="text">
      <string>Automatic Unit Conversion</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="LineEdit" name="totalLE">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ComboBox" name="unitCb"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="PushButton" name="clearPb">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckBox</class>
   <extends>QCheckBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../yoyirs.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="baseModeButtonGroup"/>
 </buttongroups>
</ui>
